Library/Homebrew/os/mac/xcode.rb

```diff
@@ -15,7 +15,7 @@ def latest_version
         when "10.8"  then "5.1.1"
         when "10.9"  then "6.2"
         when "10.10" then "7.2.1"
-        when "10.11" then "7.3.1"
+        when "10.11" then "8.0"
```

Comment:
I thought it was dodgy, but if Apple themselves are pushing it in the Xcode 8.1 release notes I'm presuming they now consider it a valid configuration. I think the situation on 10.11 was that people _should_ install the CLT if they've upgraded to Xcode 8? Misty pushed a PR on that earlier this week IIRC.